ocean_provider/routes/consume.py

```diff
@@ -91,12 +84,15 @@ def fileinfo():
     did = data.get("did")
     service_id = data.get("serviceId")
 
-    if did:
-        asset = get_asset_from_metadatastore(get_metadata_url(), did)
-        service = asset.get_service_by_id(service_id)
-        files_list = get_service_files_list(service, provider_wallet, asset)
-    else:
-        files_list = [data]
+    try:
+        if did:
+            asset = get_asset_from_metadatastore(get_metadata_url(), did)
+            service = asset.get_service_by_id(service_id)
+            files_list = get_service_files_list(service, provider_wallet, asset)
+        else:
+            files_list = [data]
+    except Exception as e:
```

Comment:
I think there is a provider wrapper that returns the error response if any Exception is encountered. Can you please check and remove this if it works as expected?

Reply:
Done. I have removed the try/except.  Exceptions are caught and handled by the `@app.errorhandler`.

Unfortunately, most of the Exceptions emitted by the provider do not include a `code` attribute so the error response uses 503 `status_code` by default. Updating the Exceptions to use more appropriate status codes is out of scope for this PR.